src/session.rs

```diff
@@ -247,10 +255,15 @@ impl BackgroundSession {
     /// Create a new background session for the given session by running its
     /// session loop in a background thread. If the returned handle is dropped,
     /// the filesystem is unmounted and the given session ends.
+    /// # Errors
+    /// Errors if the mount is poisoned.
```

Comment:
Remove this too

Reply:
I usually disable that clippy check in my other projects, because I think it's over eager. We should document errors, but only when they're part of the public API, and in that case "poisoned" needs to be explained clearly so that people understand how the mount would get into that state